testing/file_test/run_test.cpp

```diff
@@ -93,6 +93,9 @@ auto RunTestFile(const FileTestBase& test_base, bool dump_output,
     test_file.test_args = test_base.GetDefaultArgs();
     test_file.test_args.append(test_file.extra_args);
   }
+  for (const auto& include_file : test_file.include_files) {
```

Comment:
Had you considered adding these to the `fs` below too? I think that'd allow removing `SetIncludeFiles`.

Maybe:

```suggestion
  // TODO: Also add to `fs`, and remove `SetIncludeFiles`.
  for (const auto& include_file : test_file.include_files) {
```

Reply:
I was looking at that, but AddFile is in //toolchain. I can move it up to //testing

Reply:
Done